mtp_noms_ops/apps/security/forms/check.py

```diff
@@ -126,6 +126,40 @@ def get_object_list(self):
         return object_list
 
 
+class AutoAcceptListForm(SecurityForm):
+    """
+    List of AutoAccepts checks.
+    """
+    def __init__(self, request, **kwargs):
+        super().__init__(request, **kwargs)
+        self.my_list_count = 0
+
+    def get_object_list_endpoint_path(self):
+        return '/security/checks/auto-accept'
```

Comment:
no biggy, but all urls in the api end in a `/` so this will redirect on every call